Dockerfile

```diff
@@ -33,9 +33,16 @@ RUN apt-get update -qy && \
     apt-get install -y nodejs && \
     apt-get clean
 
+RUN mkdir -p /app && ln -fs /tmp /app/tmp && ln -fs /tmp /home/app
+
 COPY --from=builder /usr/local/bundle/ /usr/local/bundle/
 COPY --from=builder /app /app/
 
 WORKDIR /app
 
+RUN groupadd -g 1001 app && \
+    useradd app -u 1001 -g 1001 --home /app
```

Comment:
Did you intend to set the homedir to `/app` or `/home/app`?

Reply:
Honestly I put it there as thats what its been set as on all the other apps that have been updated for the same issue. Having another look at it though I think its probably better being `/home/app` at least then anything thats created is linked to `/tmp` rather than the `/app `